corrector_module/opmon_corrector/database_manager.py

```diff
@@ -248,22 +251,25 @@ def update_old_to_done(self, list_of_docs):
 
         return number_of_updated_docs
 
-    def check_if_hash_exists(self, doc_hash):
+    def check_clean_document_exists(self, x_request_id, document):
```

Comment:
As discussed - we could start defining type hints

Reply:
Basic type hints added